lib/src/verifiers/utilities/json-schema-verifier.ts

```diff
@@ -0,0 +1,54 @@
+import {
+  JsonSchemaType,
+  jsonTypeSchema
+} from "../../generators/contract/json-schema";
+import { TypeNode } from "../../models/nodes";
+import { DataExpression, DataType, TypeKind } from "../../models/types";
+import Ajv = require("ajv");
+
+export function verifyJsonSchema(
+  dataType: DataType,
+  data: DataExpression,
+  typeStore: TypeNode[]
+) {
+  const ajv = new Ajv();
```

Comment:
That's a fairly cryptic library name... How about naming the variable `jsonSchemaValidator`?